terminal/cursor.go

```diff
@@ -47,18 +47,24 @@ func (c *Cursor) Back(n int) error {
 
 // NextLine moves cursor to beginning of the line n lines down.
 func (c *Cursor) NextLine(n int) error {
-	if err := c.Down(1); err != nil {
+	if err := c.HorizontalAbsolute(0); err != nil {
 		return err
 	}
-	return c.HorizontalAbsolute(0)
+	if n == 0 {
+		return nil
+	}
+	return c.Down(n)
```

Comment:
Nice catch that this didn't use to forward the `n` parameter and just moved down 1 line each time 👍